docs/faq.md

```diff
@@ -45,3 +45,7 @@ The 7% fiscsal sponsorship fee will be waived for Winter Hardware Wonderland gra
 ##### Should my budget include shipping and taxes?
 
 Please do include shipping and taxes so the grant amount you receive will be as close as possible to your project expenses.
+
+##### Is there a limit on the number of people who can combine their grants?
```

Comment:
The new question you added seems to be related to https://github.com/sanjay-nagesh/winter/blob/7bdc96b0d7f8460bc50d123414c4f1a677bd338b/docs/faq.md?plain=1#L17 thoughts on just adding another line to the existing question instead of adding a new one?

Reply:
ok,i'll add to it